### ISSUE TITLE
Conflict with Worldmap

### ISSUE THREAD

**Issue by DmitrySidorow** [pR0Ps/grafana-trackmap-panel#43]
Grafana world map breaks after opening TrackMap (https://grafana.com/grafana/plugins/pr0ps-trackmap-panel)

How to reproduce it:

Worldmap settings: 
![Screenshot_4](https://user-images.githubusercontent.com/43705385/97666487-db308380-1a8e-11eb-94e8-ddc4cc5db3cb.png)

1) Worldmap works fine
![Screenshot_1](https://user-images.githubusercontent.com/43705385/97666436-bd631e80-1a8e-11eb-8370-63bc25122b1f.png)

2) Opening TrackMap
![Screenshot_2](https://user-images.githubusercontent.com/43705385/97666450-c653f000-1a8e-11eb-8ea6-8d076375da6e.png)

3) Opening WorldMap - the circle is gone and 'TypeError: Cannot read property 'x' of undefined' error
![Screenshot_3](https://user-images.githubusercontent.com/43705385/97666482-d66bcf80-1a8e-11eb-9707-a42a554b4ef9.png)

4) after re-logging in my account Worldmap works fine again:
![Screenshot_5](https://user-images.githubusercontent.com/43705385/97666680-36fb0c80-1a8f-11eb-9971-f46cdbfd6e98.png)

**Environment**:
- Grafana version: 7.3.0
- Data source type & version: postgres 12 + TimescaleDB
- OS Grafana is installed on: Ubuntu 18.04.05
- User OS & Browser: windows 10 + chrome, firefox (doesn't matter)
- Grafana plugins: Worldmap panel, TrackMap

**Comment by amotl** [comment]
Dear Dmitry,

thanks for referencing this issue at https://github.com/panodata/grafana-map-panel/issues/47#issuecomment-722166379. May I humbly suggest to use our fork and check if that works when using both a Map Panel and a Trackmap Panel on the same Dashboard? You can find release packages at [1].

With kind regards,
Andreas.

[1] https://github.com/panodata/grafana-map-panel/releases

**Comment by DmitrySidorow** [comment]
> Dear Dmitry,
> 
> thanks for referencing this issue at [panodata/grafana-map-panel#47 (comment)](https://github.com/panodata/grafana-map-panel/issues/47#issuecomment-722166379).
> 
> @amfgomez also observed that
> 
> > Grafana can not load two different map plugins at the same time.
> 
> @wetterfrosch has been observing the same thing the other day when mixing the original Grafana Worldmap Panel and our Grafana Map Panel fork. While it should work in theory, something happens under the hood which makes this thing croak. However, we have been able to confirm that multiple instances of our fork [1,2] actually work within the same dashboard.
> 
> Maybe it will also work out of the box when mixing instances of the Grafana Map Panel fork with this Grafana Trackmap Panel on the same Dashboard? You can find release packages at [3].
> 
> With kind regards,
> Andreas.
> 
> [1] https://github.com/panodata/grafana-map-panel
> [2] https://community.panodata.org/t/grafana-map-panel/121
> [3] https://github.com/panodata/grafana-map-panel/releases

yes, I can use both plugins on the same dashboard. But I want use it also in different dashboards..

**Comment by DmitrySidorow** [comment]
Just noticed. If I open this dashboard (with worldmap and trackmap) from another device, I get the same error

**Comment by amotl** [comment]
> Yes, I can use both plugins on the same dashboard. But I want use it also in different dashboards.

Seriously? I haven't been aware of that. Bummer!

> If I open this dashboard (with worldmap and trackmap) from another device, I get the same error.

Too bad. Are you already using the [grafana-map-panel](https://github.com/panodata/grafana-map-panel) within that dashboard?

**Comment by amotl** [comment]
Apart from that, and without any offense to @pR0Ps, I became aware of https://github.com/alexandrainst/alexandra-trackmap-panel by @Drazair, @j-or and contributors the other day. This apparently has been coming from [1].

While there is [2], support for Grafana 7 has been added just recently, see https://github.com/alexandrainst/alexandra-trackmap-panel/issues/18 and https://github.com/alexandrainst/alexandra-trackmap-panel/pull/24. So, you might want to use a recent release from [3] for checking that out.

[1] https://gitlab.com/synchronicity-iot
[2] https://grafana.com/grafana/plugins/alexandra-trackmap-panel
[3] https://github.com/alexandrainst/alexandra-trackmap-panel/releases

**Comment by DmitrySidorow** [comment]
> Apart from that, and without any offense to @pR0Ps, I became aware of https://github.com/alexandrainst/alexandra-trackmap-panel by @Drazair, @j-or and contributors the other day. This apparently has been coming from [1].
> 
> While there is [2], support for Grafana 7 has been added just recently, see [alexandrainst/alexandra-trackmap-panel#18](https://github.com/alexandrainst/alexandra-trackmap-panel/issues/18) and [alexandrainst/alexandra-trackmap-panel#24](https://github.com/alexandrainst/alexandra-trackmap-panel/pull/24). So, you might want to use a recent release from [3] for checking that out.
> 
> [1] https://gitlab.com/synchronicity-iot
> [2] https://grafana.com/grafana/plugins/alexandra-trackmap-panel
> [3] https://github.com/alexandrainst/alexandra-trackmap-panel/releases

[3] works great! Now I can draw path and track multiple objects. But [3] still conflicts with latest (0.13.0) grafana-map-panel in different dashboards.
